test/postsubmit-tests-with-pipeline-deployment.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 #
-# Copyright 2018 Google LLC
+# Copyright 2020 Google LLC
```

Comment:
Actually IMHO this should be the first date when this script is introduced, so I guess it should be 2018?

I'm not sure. Plz correct me if I'm wrong.